.github/workflows/java.yml

```diff
@@ -102,3 +102,121 @@ jobs:
           HERE
           # Our linters are compile-time, no point re-running tests
           mvn -P errorprone install -DskipTests
+
+  java-jni-artifacts:
+    name: "JNI Libraries/${{ matrix.platform.os }} ${{ matrix.platform.arch }}"
+    runs-on: ${{ matrix.platform.runner }}
+    strategy:
+      matrix:
+        platform:
```

Comment:
If we use `include` not `platform` here, we can use `${{ matrix.os }}` not `${{ matrix.platform.os }}`.